roq-demonstration/src/main/resources/testGCM.properties

```diff
@@ -2,12 +2,15 @@
 
 #SECTION 1 FREQUENCY
 period=3000
+# Defines the period after which the request to the GCM time out
+hcmTIMEOUT= 3000
 
 #SECTION 2 Server configuration
 #Define whether we need to format the DB on startup
 formatDB=true
 
 #SECTION 3 Cloud configuration
+hasCloudConfiguration = true
```

Comment:
What's the reason for adding this configuration item? Could we really not re-use ```cloud.use```? The answer can be "no".